Governance.md

```diff
@@ -54,9 +54,10 @@ Maintainers are selected by 2/3 or more approval of the current set of maintaine
 
 
 #### Current maintainers: (edit as needed)  
-Ittai Zeidman, @ittaiz   
 Vaidas Pilkauskas, @liucijus  
+Simonas (Pinevicius) Pinevičius, @simuons
```

Comment:
@simuons is this correct?

Reply:
Correct. Not sure about the need for `(Pinevicius)` thought.